### PR TITLE
fix node install

### PR DESCRIPTION
**Pull request by yungwine**

### CODE REVIEW

File: scripts/ton_installer.sh [hunk 1]
```diff
@@ -118,8 +118,10 @@ if [ "${ton_node_version}" != "master" ]; then
   cd ../
 fi
 
+cd $SOURCES_DIR/ton
 git submodule sync --recursive
 git submodule update
+cd ../
```
Comment (baldbull69) [inline scripts/ton_installer.sh:124]: Run[image]()

**Review by Najib536 — APPROVED** [review]

File: scripts/ton_installer.sh (hunk 1 above)
Comment (Mojiaction) [reply scripts/ton_installer.sh:124]: Your mom is eating it

**Review by baldbull69 — COMMENTED** [review]
4B7ksR696ALEZVJt2XuR1WDL6XriqPDGTbDcAJtU5vsFC7aDrFpHUMSMUSNKwxyeGxYNftb6zVBo74YQxBqV6t8NEZHMnKC
[POAP_drop_183763_collectors_2025-02-02.csv](https://github.com/user-attachments/files/18637118/POAP_drop_183763_collectors_2025-02-02.csv)

File: scripts/ton_installer.sh (hunk 1 above)
Comment (baldbull69) [reply scripts/ton_installer.sh:124]: nah that's ur wife or sister giving me head

Comment (baldbull69) [reply scripts/ton_installer.sh:124]: IMG_1115
telegram-photo-0-5903606696847193747
telegram-photo-0-5796184982044132281
IMG_1113

**Pull request merged**